test/mochaPatch.js

```diff
@@ -56,6 +56,9 @@ BbPromise.config({
   longStackTraces: true,
 });
 
+// In order to ensure that telemetry is not mistakenly emitted during test runs
+process.env.SLS_TELEMETRY_DISABLED = '1';
```

Comment:
I believe that wtih that we can also remove all `disableServerlessStatsRequests` handling (and this should be safe to remove with next `@serverless/test` major)

Reply:
that's correct, I believe it will be no longer needed :+1: